### PR TITLE
EOS:26765 - Side navigation drawer and header new UI.

### PR DESCRIPTION
**Pull request by sujit-h-singh**
# Problem Statement
- changing the design for side navigation drawer and header component.

# Design
- https://www.figma.com/file/ptVXVm0FpIIle1hZKeZqgq/LR-Theme-Redesign-Options?node-id=320%3A5798
- navigation drawer to be made an overlay and have 2 modes open and collapsed.
- changes in header section.

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by sujit-h-singh** [comment]
sidenav-collapsed, header
sidenav-open

File: gui/src/components/cortx-navigation-drawer.vue [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  .logo-title {
+    color: #6cc04a;
+    font-size: 1rem;
+  }
+}
+.font-inherit {
+  font-size: inherit;
+}
+.cortx-nav-item-active {
+  background: #262626 !important;
```
Comment (NareshGopal) [inline gui/src/components/cortx-navigation-drawer.vue:145]: It is a good practice to avoid using '!important' in CSS.

**Review by NareshGopal — CHANGES_REQUESTED** [review]

File: gui/vue.config.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
 module.exports = {
     outputDir: 'web-dist',
     devServer: {
-        proxy: 'http://localhost:28100'
+        proxy: 'https://localhost:28100'
```
[the remark this answers is not in the document]
Comment (sujit-h-singh) [reply gui/vue.config.js:20]: reverting it.

File: gui/src/components/cortx-navigation-drawer.vue (hunk 1 above)
Comment (sujit-h-singh) [reply gui/src/components/cortx-navigation-drawer.vue]: removed.

**Review by NareshGopal — APPROVED** [review]

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**